src/Nancy/Conventions/BuiltInCultureConventions.cs

```diff
@@ -133,7 +133,12 @@ public static CultureInfo CookieCulture(NancyContext context, GlobalizationConfi
         /// <returns>CultureInfo from CurrentThread</returns>
         public static CultureInfo ThreadCulture(NancyContext context, GlobalizationConfiguration configuration)
         {
+#if DOTNET5_4
+            CultureInfo.CurrentCulture = new CultureInfo(configuration.SupportedCultureNames.First());

```

Comment:
What I meant it, is setting the `CultureInfo.CurrentCulture`, to the first `SupportedCultureNames`, and then returning it the same as returning `Thread.CurrentThread.CurrentCulture`?

Before we just returned `Thread.CurrentThread.CurrentCulture` which was set by something else based on some criterion, now we explicitly set it to the first configured one'

I know you can change it using

```xml
<globalization culture="en-US" />
```

in your `web.config`, which would mimic using `SupportedCultureNames.First()`, but if the `globalization` element then did it not pick it up form the OS? That wouldn't be a supported case for us anymore, with this code, and a major breaking change?